conduce.py

```diff
@@ -22,6 +23,25 @@ def list_from_args(args):
     return api.list_resources(object_to_list.upper().rstrip('S'), **vars(args))
 
 
+def create_resource(args):
+    resource_type = args.type.upper()
+    resource_name = args.name
+    content_path = args.content
+
+    del vars(args)['name']
```

Comment:
does this work?  I'd assume that vars(args) created a new variable, and name was deleted from that.  I think this should be something like `kwargs = vars(args); del kwargs['name'] ...`

Reply:
looks like it works with python 2 at least

Reply:
It works.  I use that pattern throughout.